deploy/src/main/sandbox-k8s/30-deploy-hono.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#*******************************************************************************
+# Copyright (c) 2021 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+# This script deploys Hono to the Eclipse Foundation's sandbox at hono.eclipseprojects.io
+# It requires that the namespace "hono" already exists and that the secret for the certificate is present.
+
+set -ue
+
+NS_HONO=hono
+KUBECONFIG=${2:-"--kubeconfig /etc/rancher/k3s/k3s.yaml"}
```

Comment:
can we source the shared variables from a common file?

Reply:
Yes, makes sense. Regarding the name of the namespace "hono" I tend to inline it. It is part of other files like the certificate definition, so making it configurable will be hard and not necessary.

Reply:
I have changed it.